Packs/CommonPlaybooks/Playbooks/playbook-Email_Address_Enrichment_-_Generic_v2.1.yml

```diff
@@ -714,22 +718,18 @@ inputs:
 - key: Domain
   value:
     complex:
-      root: inputs.Domain
-      filters:
-      - - operator: isNotEmpty
-          left:
-            value:
-              simple: inputs.Domain
-            iscontext: true
+      root: Account.Email
       transformers:
       - operator: uniq
+      accessor: Domain
```

Comment:
R u sure that you want to add all the domains (such as URL's or any extracted domains) from each incident?

Reply:
removed it